gazelle/buf/repositories.bzl

```diff
@@ -1,19 +1,28 @@
 """Dependencies required to use buf gazelle extension."""
 
 load("@bazel_gazelle//:deps.bzl", "go_repository")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 def gazelle_buf_dependencies():
     """Utility method to load dependecies of the buf gazelle extension"""
 
     go_repository(
         name = "com_github_bazelbuild_bazel_gazelle",
         importpath = "github.com/bazelbuild/bazel-gazelle",
-        sum = "h1:gPaI2C0hRAhNT4w3bdSCift3nFjq0j07I4bmSDeneA4=",
-        version = "v0.24.0",
+        sum = "h1:umlCZxzCEr8BHdesfY2sv5T8NTv9+JiPL8f/Vk83Aag=",
+        version = "v0.25.0",
     )
     go_repository(
         name = "in_gopkg_yaml_v3",
         importpath = "gopkg.in/yaml.v3",
-        sum = "h1:h8qDotaEPuJATrMmW04NCwg7v22aHH28wwpauUhK9Oo=",
-        version = "v3.0.0-20210107192922-496545a6307b",
+        sum = "h1:fxVm/GzAzEWqLHuvctI91KS9hhNmmWOoWu0XTYJS7CA=",
+        version = "v3.0.1",
+    )
+    http_archive(
+        name = "com_github_bazelbuild_buildtools",
+        sha256 = "ae34c344514e08c23e90da0e2d6cb700fcd28e80c02e23e4d5715dddcb42f7b3",
+        strip_prefix = "buildtools-4.2.2",
+        urls = [
+            "https://github.com/bazelbuild/buildtools/archive/refs/tags/4.2.2.tar.gz",
+        ],
```

Comment:
Why do we use `go_repository` for the other golang dependencies but http_archive for this one? Are there special update instructions (other than the gazelle update-repos for the `go_repository` ones)?

Reply:
I already had that as a `http_archive` and gazelle didn't generate a `go_repository` rule, I thought it was a special case. Now that I remove the build tools and run it again I see it pop up. I'll change this to `go_repository`